src/ExpressiveInstaller/Resources/templates/plates/home-page.phtml

```diff
@@ -63,7 +63,7 @@
         <p>
             Expressive promotes and advocates the usage of Dependency Injection/Inversion of Control containers
             when writing your applications. Expressive supports multiple containers which typehints against
-            <a href="https://github.com/container-interop/container-interop" target="_blank">container-interop</a>.
+            <a href="https://github.com/php-fig/container" target="_blank">container-interop</a>.
```

Comment:
So I think link text should be also updated to "PSR Container" (instead of "container-interop").

Reply:
done ;)